frontend/src/Project/Canvas/LabeledCanvas.js

```diff
@@ -71,6 +73,7 @@ export const LabeledCanvas = ({ setBitmaps }) => {
         output: [width, height],
         graphical: true,
         dynamicArguments: true,
+        loopMaxIterations: 5000,
```

Comment:
Can you add a comment mentioning that this controls the max number of cells on the canvas?